packages/core/src/base-chart.ts

```diff
@@ -363,6 +367,32 @@ export class BaseChart {
 		resizeObserver.observe(this.holder);
 	}
 
+	/**
+	 * Draws title within the Chart's svg element.
+	 * @param title
+	 */
+	drawTitle(title: string) {
+		if (title) {
+			this.svg
+			.append("text")
+			.classed("chart-title", true)
+			.text(title);
+		}
+		const titleMargin = Configuration.charts.marginForChartTitle;
+
+
+		// retrieve the current transformations (specific to each chart type)
+		const transform = this.innerWrap.attr("transform");
+		const translateArr = transform.substring(transform.indexOf("(") + 1, transform.indexOf(")")).split(",");
```

Comment:
**Do we absolutely need this?**
**If no =>** what's a more readable way to do this?
**If yes =>** this should become a function in `tools.ts` that can parse transform strings, and give you certain values back

Reply:
Yes, so true, this should be in tools.  We need this because specific chart types are translated already (donut/pie) since they are drawn from the center. Adding an optional title we need to translate the chart more but not undo the current translations that center the chart.